lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -85,6 +85,22 @@ llvm::Error HostManager::addNetwork(std::unique_ptr<Module> module,
                           name);
     }
   }
+
+  if (cctx.precisionConfig.quantMode == QuantizationMode::Profile) {
```

Comment:
Is it possible to tweak this outside of the host manager? May be in the client of the host-manager like image-classifier or something?

The reason I ask is that I'm a bit worried about making HostManager aware of quantization.

Reply:
"Is it possible to tweak this outside of the host manager? May be in the client of the host-manager like image-classifier or something?" 
-- It is Host Manager calls the different partition method, so I don't think it is a good way to make this decision outside. 
But I think we can let the Partitioner to make this decision. Something like if we would like to skip the backendBasedPartition or not.

Reply:
> But I think we can let the Partitioner to make this decision. Something like if we would like to skip the backendBasedPartition or not.

Sounds reasonable. This would be better than having both HostManager and Partitioner being aware of it.